filebeat/tests/system/test_input.py

```diff
@@ -520,6 +520,7 @@ def test_harvester_limit(self):
             harvester_limit=1,
             close_inactive="1s",
             scan_frequency="1s",
+            logging_level="debug"
```

Comment:
I think this can be removed and it should still work.

Reply:
removed